op-e2e/opgeth/op_geth.go

```diff
@@ -62,7 +62,9 @@ func NewOpGeth(t testing.TB, ctx context.Context, cfg *e2esys.SystemConfig) (*Op
 
 	var allocsMode genesis.L2AllocsMode
 	allocsMode = genesis.L2AllocsDelta
-	if graniteTime := cfg.DeployConfig.GraniteTime(l1Block.Time()); graniteTime != nil && *graniteTime <= 0 {
+	if holoceneTime := cfg.DeployConfig.HoloceneTime(l1Block.Time()); holoceneTime != nil && *holoceneTime <= 0 {
```

Comment:
Do we want to unify the logic to choose `allocsMode` in e2e test with [here](https://github.com/ethereum-optimism/optimism/blob/fdab9b01e3aed6329ac61bea5ce96b1050e91408/op-e2e/e2eutils/setup.go#L140) ?

Reply:
great suggestion, done.